app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -18,7 +18,7 @@ public function __construct(private AuthManager $authManager) {}
     public function handle(Request $request, \Closure $next, ?string $guard = null): mixed
     {
         if ($this->authManager->guard($guard)->check()) {
-            return redirect()->route('index');
+            return redirect()->route('filament.app.resources...index');
```

Comment:
Redirect to `\App\Filament\App\Resources\ServerResource\Pages\ListServers::getUrl()` instead of a route.